File: level2/javascript/exercise5_easy.js
// Від трицифрового числа N взяли першу цифру, 
// прибрали її та додали в кінець числа. Наприклад, якщо N = 146:
// 146 —> 461 Знайдіть отримане число Р.
// Формат вхідних даних. На вхід подається одне натуральне трицифрове число N.
// Формат вихідних даних. Виведіть число P.

function rotateNumber(n) {

}


const testCases = [146, 802, 375];
const expectedResults = [461, 28, 753];

function testRotateNumber() {
    const startTime = performance.now();
    testCases.forEach((test, index) => {
        const result = rotateNumber(test);
        console.assert(result === expectedResults[index], `Test case ${test} failed: expected ${expectedResults[index]}, got ${result}`);
    });
    const endTime = performance.now();
    console.log(`All JavaScript tests passed! Execution time: ${(endTime - startTime).toFixed(6)} milliseconds`);
}

testRotateNumber();

File: level2/python/exercise5_easy.py
# Від трицифрового числа N взяли першу цифру, 
# прибрали її та додали в кінець числа. Наприклад, якщо N = 146:
# 146 —> 461 Знайдіть отримане число Р.
# Формат вхідних даних. На вхід подається одне натуральне трицифрове число N.
# Формат вихідних даних. Виведіть число P.

# def rotate_number(n):



test_cases = [146, 802, 375]
expected_results = [461, 28, 753]


import time
def test_rotate_number():
    start_time = time.time()
    for i, test in enumerate(test_cases):
        result = rotate_number(test)
        assert result == expected_results[i], f"Test case {test} failed: expected {expected_results[i]}, got {result}"
    end_time = time.time()
    print(f"All Python tests passed! Execution time: {end_time - start_time:.6f} seconds")

test_rotate_number()